.github/ISSUE_TEMPLATE/2-vscode_extension_bug.yml

```diff
@@ -0,0 +1,143 @@
+name: Report a Bug With the "Salesforce Code Analyzer" VS Code Extension
+description: Report an issue that's with the VS Code extension but not with the CLI commands.
+title: "[BUG][vscode-extension] <YOUR_TITLE_HERE>"
+body:
+  - type: checkboxes
+    attributes:
+      label: Have you tried to resolve this issue yourself first?
+      description: |
+        You can often resolve "vscode-extension" issues on your own. Follow these steps:
```

Comment:
"vscode-extension" feels wrong, but I'm not intuitively sure what should go here. Should it be the name of the actual pacakge the VSCode extension is released as, maybe?

Reply:
I'll go with a generic! that's what I did for the gha